tools/tx_pull.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+
+"""
+This script is meant to pull the translations from Transifex .
+Technically, it will pull the translations from transifex,
+compare it with the po files in the repository and replace it if needed
+"""
```

Comment:
Could you please document here how to install its dependencies (Transifex), how to configure it and how use it ?